README.md

```diff
@@ -1,34 +1,16 @@
 # Descriptive-Data-Uploading-App
 
-## Set up the application
+Whenever it exports the zip folder from your source system, have your custom export app automatically run the DescriptiveDataUploadApp. Clone the DescriptiveDataUploadApp to your machine by running the following command: `git clone https://github.com/microsoft/vivainsights_ingressupload.git`.
 
-1.	Clone the app. Open a command prompt and enter the following command: `git clone https://github.com/microsoft/vivainsights_ingressupload.git`.
-2.	If Visual Studio was open, close it. Open or re-open Visual Studio as an administrator.
-3.	On the right, select Open a local folder. Choose the cloned folder (vivainsights_ingressupload). 
-   
-    **Note:** The cloned folder will live in whichever directory you ran the git clone command from.
-   
-4.	On the right, in the **Solution Explorer** tab, double-click **DescriptiveDataUploadApp.sln**.
 
-      <img width="350" alt="admin-upload-app-sln" src="https://user-images.githubusercontent.com/98846621/229250984-54df60e1-5249-4cd2-9f04-6272d63143a1.png">
-      
-5.	At the top of Visual Studio, you’ll need to select a start-up project. Select **DescriptiveDataUploadApp.csproj**.
+**Note:** If you haven't downloaded the zip folder yet, find it [here](https://go.microsoft.com/fwlink/?linkid=2230444). Refer to our [main data-import documentation](https://learn.microsoft.com/viva/insights/advanced/admin/import-org-data-first#prepare-the-data-export) for more information about using the files in this folder.
 
-6.	Select the play button to run the app or press Ctrl + F5 on your keyboard. 
+After the custom export app runs the DescriptiveDataUploadApp, a console pops up asking you for the following inputs: 
```

Comment:
We don't need to mandate that their custom export app runs. Same reasoning as above comment.